docs/guide.md

```diff
@@ -0,0 +1,445 @@
+## Setup
```

Comment:
This file is big and it is hard to understand the organization of the guide.

I think either splitting it or adding a TOC would be good.

In Sublime text, there is a plugin that can generate a TOC for a markdown file automatically ;)

Reply:
Yes, definitely a next step as mentioned in #57 .